### ISSUE TITLE
How to load ietf-routing on Netconfd

### ISSUE THREAD

**Issue by vggonzalez** [OpenClovis/OpenYuma#38]
Hi everyone,

After reading the whole documentation it's unclear to me how I can load one module (ietf-routing in this case) into the netconfd server.

I know the command is "$netconfd --module=<desiredModule>", my question is: The desired module is a path to the .yanf file that I have to create, right? 

Also, one more question: after connecting to this netconfd server through a SDN Controller (ONOS in my case) and editting the configuration stablished by ietf-routing (next hop, for example)... Would that change affect the real device's configuration or is it just a test server with no real influence on the device's configuration?

Regards

**Comment by hungta** [comment]
To gain your purpose, you must use our product called "SAFplus-Mgt" which generates your .yang file to C++ source code and you just simply build the app and run.

**Comment by hungta** [comment]
1. Yes,  --module=... which includes full path or the yang file will be searched in env $YUMA_MODPATH
2. "Editing configuration stablished by ietf-routing (next hop, for example)..." do you mean the yang file?

**Comment by vggonzalez** [comment]
Yes. YANG defines a data model for the device which I can edit through a NETCONF session. 
However, unless I write a .c module that connects the .yang file with the device's real configuration, any change on the device wouldn't affect it's real behaviour as it would be just a NETCONF test session.
I will have to write this .c module and then compile it into a .so, however I'm looking at your module ietf-interfaces.c and it's unclear to me how I can translate that into ietf-routing.c

**Comment by hungta** [comment]
To gain your purpose, you should try our product called "SAFplus-Mgt" which generates your .yang file to C++ source code and you just simply build and run it.
SAFplus-Mgt also contains tools for simply interacting with netconf server (OpenYuma).
